app/code/Magento/BundleGraphQl/Model/Resolver/Options/BundleEnteredOptionValueIdV2.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\BundleGraphQl\Model\Resolver\Options;
+
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\Framework\GraphQl\Query\Resolver\ContextInterface;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+
+/**
+ * Format new option id_v2 in base64 encode for entered bundle options
+ */
+class BundleEnteredOptionValueIdV2 implements ResolverInterface
```

Comment:
Implementation looks neat. The name here BundleEnteredOptionValueIdV2 is little confusing because a bundle option is something that need not be entered. lets just keep it simple and name it BundleItemOptionIdV2, the same hierarchy it follows in the schema.

Reply:
Agree on that, the new name seems much better. However, we may also have the possibility to enter the option's quantity, which makes the quantity an entered value (I've also raised this question in slack channel).
Thank you for the suggestion.